static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -320,15 +319,12 @@ const makeABTestVariant = (
                 locations.some(
                     region => geolocationGetSync() === region.toUpperCase()
                 );
-            const matchesKeywordIds =
-                keywordIds.length === 0 ||
-                keywordIds.some(keywordId =>
-                    config.get('page.keywordIds').includes(keywordId)
-                );
-            const matchesToneIds =
-                toneIds.length === 0 ||
-                toneIds.some(toneId =>
-                    config.get('page.toneIds').includes(toneId)
+            const matchesTags =
+                tagIds.length === 0 ||
+                tagIds.some(tagId =>
+                    `${config.get('page.keywordIds')},${config.get(
+                        'page.nonKeywordIds'
```

Comment:
`nonKeywordTagIds`?
https://github.com/guardian/frontend/blob/91b63dd270f33a56b9ccbe7e5b59dab6dff94317/common/app/model/meta.scala#L800

Reply:
great spot! I put the PR before testing it...! 👀